sovtoken/setup.py

```diff
@@ -39,7 +39,8 @@
     include_package_data=True,
 
 
-    install_requires=['indy-plenum-dev>=1.6.733'],
+
+    install_requires=['indy-node-dev>=1.6.858'],
```

Comment:
1.6.858 version of indy-node depends on indy-plenum 1.6.721 (https://github.com/hyperledger/indy-node/blob/1.6.858-master/setup.py#L59).
Please use the latest indy-node instead which is 1.6.874.